tools/db_crashtest.py

```diff
@@ -888,6 +869,23 @@ def finalize_and_sanitize(src_params):
         dest_params["metadata_write_fault_one_in"] = 0
         dest_params["read_fault_one_in"] = 0
         dest_params["metadata_read_fault_one_in"] = 0
+        if dest_params.get("txn_write_policy", 0) != 0:
+            # TODO: should any of this change for WUP (txn_write_policy==2)?
```

Comment:
cc @cbi42 if there is nothing obvious standing out to you, I'm fine with letting the stress test to find us option incompatibility.

Reply:
> letting the stress test to find us option incompatibility

Not sure if multi_ops_wup was intentionally not enabled, but this sounds good to me.